logdetective/constants.py

```diff
@@ -0,0 +1,29 @@
+
+# pylint: disable=line-too-long
+DEFAULT_ADVISOR = "https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/resolve/main/mistral-7b-instruct-v0.2.Q4_K_S.gguf?download=true"
+
+PROMPT_TEMPLATE = """
+Given following log snippets, and nothing else, explain what failure, if any occured during build of this package.
```

Comment:
If this written as one sentence on purpose? For me the sentence structure is harder to understand.

I would split it to ``... failure. If any occurred during...``

Also thinking if `failure` is covering everything? I wonder if `error` is not better.

Reply:
Yes. It's not really for people, it's for the model. But it is missing a comma. We can't really put `error` in there, as that could make the model less inclined to consider anything else than messages including `error` substring.